src/Components/Form/FormFields/PhoneNumberFormField.tsx

```diff
@@ -73,16 +54,18 @@ export default function PhoneNumberFormField(props: Props) {
 
   const setValue = (value: string) => {
     value = value.replaceAll(/[^0-9+]/g, "");
-
-    asYouType.reset();
-    asYouType.input(value);
+    if (value.length > 12 && value.startsWith("+910")) {
+      value = "+91" + value.slice(4);
+    }
```

Comment:
What does this intend to do? Can you add comments for the same.

Reply:
It eliminates 0 if any at the beginning of a number after +91. For ex +9109876543214 will be formatted as +919876543214. The libphonenumber-js library was also doing this. Hyderabad's area code for landline is 40 but I have never seen landline number with 40, everyone uses 040 followed by 8 digits. Similarly for other areas. That 0 should be eliminated while storing it in E164 format. Later I also noticed that chrome's autofill feature adds a 0 at the beginning after country code. So this will handle that too.